hw/ip/usbdev/pmod/dualpmod-kicad/dualpmod.sch

```diff
@@ -41,8 +41,8 @@ F 0 "R4" V 3695 2300 50  0000 C CNN
 F 1 "22R" V 3786 2300 50  0000 C CNN
 F 2 "Resistor_SMD:R_0805_2012Metric_Pad1.15x1.40mm_HandSolder" H 3900 2300 50  0001 C CNN
 F 3 "~" H 3900 2300 50  0001 C CNN
-	1    3900 2300
-	0    1    1    0   
+  1    3900 2300
+  0    1    1    0   
```

Comment:
Are you sure these files still open correctly without tabs?

Reply:
I am not. Any idea how to check? I will restore this file and re-push.